Schema docs edit (#276)
seqeralabs/nf-tower-docs: docs/pipeline-schema/overview.md

## Overview

> This page will give you a detailed description of what pipeline schema files are, why they are used and to give you an in-depth description of how to build and customize your own Pipeline JSON schema file.

Reviewer: ```suggestion
This page will give you a detailed description of what pipeline schema files are, why they are used, and show you how to build and customize your own Pipeline JSON schema file.
```

![Tower Launch interface](_images/pipeline_schema_form.png)

## What is a Pipeline schema?

Pipeline schema files describe the structure and validation constraints of your workflow parameters. They are used to validate parameters before launch to prevent software/pipelines failing in unexpected ways at runtime.

Tower uses pipeline schema to build the launchpad parameters form.

!!! tip
    You can populate the parameters in the pipeline, by uploading a YAML or a JSON file, in addition to filling it on the UI itself.

## How can I build my own Pipeline schema file for my Nextflow pipelines?

> The pipeline schema is based on [json-schema.org](https://json-schema.org/) syntax, with some additional conventions. It can quickly get very complex so do not recommend creating or editing this file manually.

Reviewer: ```suggestion
The pipeline schema is based on [json-schema.org](https://json-schema.org/) syntax, with some additional conventions. This can get complex, so we do not recommend creating or editing this file manually.
```

Author: I'd recommend grouping comments together into a single review - then everyone gets one email in total instead of one per comment 😄 (apologies for already merging your other suggested change - I'll wait a bit now! 😅 )

Reviewer: Hi Phil, I committed all my suggestions in one go and merged the PR thereafter. Not sure why it triggered multiple emails, apologies for the unintended spam.

Author: Yup, no worries - the commits were batched 👍🏻  It looks like you submitted each suggested change as a separate review + comment, so I got an email for each separately. Instead, you can save them to the end and submit once, which sends them all through in one go.
